transforms/add-layout-property/index.js

```diff
@@ -0,0 +1,47 @@
+const { getParser } = require('codemod-cli').jscodeshift;
+const { getOptions } = require('codemod-cli');
+
+const addLayoutImport = (j, root, layoutPath) => {
+  let allImports = root.find(j.ImportDeclaration);
+  let oldImport = allImports.find(j.ImportDefaultSpecifier, {
+    local: { name: 'layout' },
+  });
+  oldImport.closest(j.ImportDeclaration).remove();
```

Comment:
Why do we need to remove the old layout import?
And why are we finding again all import declarations and assigning it to `allImports`?

Reply:
Updated the logic to update the import path if the layout import already exists.